packages/runtime/container-runtime/src/blobManager.ts

```diff
@@ -527,7 +555,7 @@ export class BlobManager {
      * When running GC in test mode, this is called to delete blobs that are unused.
```

Comment:
Update this comment to add tombstone info. Maybe look at the comment in datastores.ts for this function.